### DISCUSSION TITLE
xdrip watch does not work with wearos 4

### DISCUSSION THREAD

**Discussion by corneliusroemer** [NightscoutFoundation/xDrip#3137] [Bug reports]
I've tried to side load the xdrip+ app using wear installer 2 on my pixel watch 2 via my pixel 8 Pro. They came out of the box with wear os 4 and android 14.

It does not work as wear installer does not show xdrip in the list of apps with wear components.

It seems that the xdrip APK is incompatible with wearos 4?

This would explain the issues other people have reported with android 14 and wear os 4 on Samsung after upgrading.

Related discussions:
https://github.com/NightscoutFoundation/xDrip/discussions/3003
https://github.com/NightscoutFoundation/xDrip/discussions/3126

**Comment by kylechase** [comment]
Did you follow the instructions for installing apps on WearOS 4 from wear installer?
https://www.youtube.com/watch?v=abgN4jQqHb0
